distribution/docs/sender/application.properties

```diff
@@ -37,6 +37,18 @@ db-event.retry.initial.delay=120000
 
 # Number of events to sync in parallel, defaults to 10
 events.sync.parallel.size=
+
+# The interval in milliseconds before the first sender sync messages fetch by the active mq publisher route when the application starts.
+active-mq-publisher.initial.delay=120000
+
+# The interval in milliseconds between sender sync messages fetches by the active mq publisher route.
+active-mq-publisher.delay=60000
+
+# The interval in milliseconds before the first sender sync responses fetch by the sync response processor route when the application starts.
+sync-response-processor.initial.delay=120000
```

Comment:
2 minutes is a long time for initial delay

Reply:
What delay should we set?